src/encoded/schemas/mixins.json

```diff
@@ -133,6 +133,67 @@
             "permission": "import_items"
         }
     },
+    "review": { 
+        "review": {
+            "title": "Characterization review",
+            "description": "Review of the characterization results by an independent reviewer.",
+            "type": "object",
+            "additionalProperties": false,
+            "dependencies": {
+                "review_status": {
```

Comment:
Here it is called `review_status` but in the properties it is called `status`? This works?

Reply:
fixed. now it is called status everywhere